proto/cosmos/authz/v1beta1/tx.proto

```diff
@@ -32,8 +33,7 @@ message MsgGrant {
   string granter = 1;
   string grantee = 2;
 
-  google.protobuf.Any       authorization = 3 [(cosmos_proto.accepts_interface) = "Authorization"];
-  google.protobuf.Timestamp expiration    = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  cosmos.authz.v1beta1.Grant grant = 3 [(gogoproto.nullable) = false];
```

Comment:
So the idea is that if we add other fields to `grant` they get included automatically? I guess that makes sense right?

Reply:
Yes